docs/source/item-validation.rst

```diff
@@ -184,6 +212,36 @@ A `list` containing the location of the item schema. Could be a local path or a
 If you are working on a spider that produces multiple items types, you can define it
 as a `dict`:
 
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_SCHEMAS: {
+        DummyItem: '/path/to/dummyitem_schema.json',
+        OtherItem: '/path/to/otheritem_schema.json',
+    }
+
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the local path of the item schema.
+
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_CERBERUS: [
+        '/path/to/schema.json',
+        'https://api.myjson.com/bins/gmdgl'
```

Comment:
I prefer an fake URL (like `http://example.com/mycerberusschema`)

Reply:
Generalized docs, I like it. 
Implemented.